src/assets/i18n/uk.json

```diff
@@ -735,7 +735,7 @@
   "DNS Timeout": "",
   "Do any of them look similar?": "",
   "Do not connect to a fibre channel port": "",
-  "Do not enable ALUA on TrueNAS unless it is also supported     by and enabled on the client computers. ALUA only works when enabled     on both the client and server.": "",
+  "Do not enable ALUA on TrueNAS unless it is also supported       by and enabled on the client computers. ALUA only works when enabled       on both the client and server.": "",
```

Comment:
Extra spaces added.

Reply:
And in other places where there's a long string, it's the same.

Reply:
Good catch.